pyproject.toml

```
@@ -0,0 +1,108 @@
+[project]
+  name="array-api-typing"
+  dynamic = ["version"]
+  description = "Static typing support for the array API standard"
+  readme = "README.md"
+  requires-python = ">=3.10"
+  license.file = "LICENSE"
+  authors = [
+    {name="Consortium for Python Data API Standards", email="data-apis@users.noreply.github.com"},
+    {name="Joren Hammudoglu", email="jhammudoglu@gmail.com"},
+    {name="Nathaniel Starkman", email="nstarman@users.noreply.github.com"}
+  ]
+  classifiers = [
+    "Development Status :: 3 - Alpha",
```

I suppose that once we publish the first version, we'll be calling it an "alpha".
But perhaps the most "agile" thing would be to ignore the future and use something that's closer to our current project status, such as

```suggestion
    "Development Status :: 1 - Planning",
```

So I guess it's mostly personal preference, and in this case I don't have one 🤷🏻

---

Done.